### DISCUSSION TITLE
Photoprism fails to import .mp4 files | Videos

### DISCUSSION THREAD

**Discussion by leofabri** [tteck/Proxmox#225] [General]
I'm getting a lot of errors when importing .mp4 files:

`2022-05-07 20:14:34 ERRO import: file type mp4 not supported in 2022/05/20220507_181433_87DE35D0.mp4 (convert to jpeg)`
Those files are also never listed inside the Video category.

To my understading, mp4s files should be supported by photoprism itself. Is it working for you?

**Comment by tteck** [comment]
I personally have never used PhotoPrism.

**Reply by leofabri** [reply]
Really? haha I thought you had all of these softwares

**Comment by tteck** [comment]
I made a few more changes

**Reply by leofabri** [reply]
Time for an update I guess
